packages/next/test/fixtures/00-app-dir-segment-cache/vercel.json

```diff
@@ -18,6 +18,15 @@
         "Next-Router-Segment-Prefetch": "/_tree"
       },
       "mustContain": "foobar-1"
+    },
+    {
+      "path": "/api/status",
+      "status": 200,
+      "responseHeaders": {
+        "x-vercel-cache": "MISS",
+        "cache-control": "public, max-age=0, must-revalidate"
+      },
+      "mustContain": "\"status\":\"ok\""
```

Comment:
This test case passes without the changes here

<img width="2076" height="1328" alt="CleanShot 2025-08-15 at 11 09 51@2x" src="https://github.com/user-attachments/assets/a477f2be-2fff-4b78-bef7-1b0e0bca4514" />